Maths/test/WhileLoopFactorial.test.js

```diff
@@ -1,12 +1,12 @@
 import { factorialize } from '../WhileLoopFactorial'
 
 function testFactorial(n, expected) {
```

Comment:
You were supposed to add the space here, after `testFactorial`, not after all function calls...